src/symbols_linux.cpp

```diff
@@ -145,22 +145,29 @@ typedef Elf32_Dyn  ElfDyn;
 
 #if defined(__x86_64__)
 #  define R_GLOB_DAT R_X86_64_GLOB_DAT
+#  define R_DIRECT R_X86_64_64
 #elif defined(__i386__)
 #  define R_GLOB_DAT R_386_GLOB_DAT
+#  define R_DIRECT -1
 #elif defined(__arm__) || defined(__thumb__)
 #  define R_GLOB_DAT R_ARM_GLOB_DAT
+#  define R_DIRECT -1
 #elif defined(__aarch64__)
 #  define R_GLOB_DAT R_AARCH64_GLOB_DAT
+#  define R_DIRECT -1
```

Comment:
Isn't there a AArch64 equivalent?
I see it generates `R_AARCH64_ABS64`

Reply:
I originally added something for all archs, but given I cannot test them, I focused on x86-64. I can add it for arm64.